src/ray/raylet/node_manager.cc

```diff
@@ -1600,9 +1726,48 @@ void NodeManager::HandleRequestWorkerLease(const rpc::RequestWorkerLeaseRequest
             reply->mutable_worker_address()->set_port(worker->Port());
             reply->mutable_worker_address()->set_worker_id(worker->WorkerId().Binary());
             reply->mutable_worker_address()->set_raylet_id(self_node_id_.Binary());
+            RAY_LOG(WARNING) << "leased_worker 2 = " << worker->WorkerId();
             RAY_CHECK(leased_workers_.find(worker->WorkerId()) == leased_workers_.end());
             leased_workers_[worker->WorkerId()] = worker;
-            leased_worker_resources_[worker->WorkerId()] = request_resources;
+// TODO (Ion): Fix handling floating point errors, maybe by moving to integers.               
+#define ZERO_CAPACITY 1.0e-5
```

Comment:
Please make this a constant, such as `const kZeroCapacity` and document what it's used for.